earthaccess/search.py

```diff
@@ -1,16 +1,60 @@
 import datetime as dt
 from inspect import getmembers, ismethod
-from typing import Any, Dict, List, Optional, Tuple, Type, Union
+from typing import Any, List, Optional, Tuple, Type, Union
 
 import dateutil.parser as parser  # type: ignore
-from cmr import CollectionQuery, GranuleQuery  # type: ignore
-from requests import exceptions, session
+from cmr import CollectionQuery, GranuleQuery
+from requests import exceptions, get, session
```

Comment:
I suggest _not_ importing `get`, and instead referring to it as the qualified name `requests.get` elsewhere, especially since you have locally defined `get` method in this file, which may lead to a bit of confusion.